torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2003,13 +2003,6 @@ def sample_inputs_bernoulli(self, device, dtype, requires_grad, **kwargs):
                         requires_grad=requires_grad)
         yield SampleInput(t)
 
-def error_inputs_bernoulli(op_info, device, **kwargs):
```

Comment:
Why was this test removed ?

Reply:
bernuoulli uses random_mps_impl under the hood so the error wasn't valid anymore since it support non-contiguous tenors now too. So I removed it but on second thought, it might be useful to keep the test as a passing test for documentation.

I added it back to the sample_inputs_bernoulli method. Open to other ideas too!

Reply:
Hi @kulinseth - what do you think?

Reply:
Thanks.